widgets/js/index.tsx

```diff
@@ -66,6 +87,58 @@ function renderChart({ model, el }: RenderArgs) {
   model.on("change:estimates", onChange);
 }
 
+function renderScatterChart({ model, el }: RenderArgs) {
+  const onChange = () => {
+    const results = model.get("estimates");
+
+    const estimates = [];
+    if (results[0] == null) {
+      estimates.push(results);
+    } else {
+      for (const estimate of Object.values(results)) {
+        estimates.push(estimate);
+      }
+    }
+
+    (estimates as Array<any>).forEach(
+      (item, idx) =>
+        (item.jobParams.runName = `(${String.fromCharCode(0x61 + idx)})`),
```

Comment:
Wouldn't this start to look a bit weird after `idx` exceeds 26? Is it important to use letters or can we use a number suffix?

Reply:
Good question! Thank you, @minestarks ! On one hand, the chart and the table could show many dozens of series and rows correspondingly. We even prepare colormap for the purpose. However, it is pretty hard to analyze the chart and the summary table with the number of series/rows greater than 10-15, ok 20. So, technically we could replace (a), (b), (c) with (1), (2), (3). But do we want to support such scenarios? Or maybe we could show (a), (b), (c) if the number of series/rows is less than 27, and numbers otherwise.

We have the same code on VS Code extension side (this one is a copy for Jupyter widgets).
@billti and @minestarks , what do you think?

Reply:
Here are my thoughts:
1. for VS Code scenario labels like "BitFlip (a)", "BitFlip (b)" are enough because we would consider a few items in the run (now just 8) and add "-2", "-3" for next runs.
2. For Jupyter notebook scenario, it is worth to have numbers (not letters for easiest correspondence with params sent for the estimation). I have added this starting with 1. Please let me know if you think we should start with 0.
3. For Jupyter scenario, it could be helpful to show custom labels for runNames. I have added this as well and provided in a sample.